File: VonexDealer.API/Models/Order/Churn.cs
using Newtonsoft.Json;
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace VonexDealer.API.Models.Order
{
    public class Churn
    {
        public Int64 ChurnID { get; set; }
        public Int64 LandlineID { get; set; }
        public string ContractPeriod { get; set; }
        public string ServiceNumber { get; set; }
        public Int64 RatePlanID { get; set; }
        public Int64 AddressID { get; set; }
      [JsonIgnore]
      [NotMapped]
        public string FormattedAddress { get; set; }

    }
}

File: VonexDealer.Vue/ClientApp/models/Churn.js
export class Churn {
    constructor({
        churnID = 0,
        landlineID = 0,
        contractPeriod = null,
        serviceNumber = null,
        ratePlanID = null,
        addressID = 0
    } = {}) {
        this.churnID = churnID;
        this.landlineID = landlineID;
        this.contractPeriod = contractPeriod;
        this.serviceNumber = serviceNumber;
        this.ratePlanID = ratePlanID;
        this.addressID = addressID;
    }

}
export function createChurn(data) {
    return Object.freeze(new Churn(data));
}
